widgetry/src/style/button_style.rs

```diff
@@ -0,0 +1,135 @@
+use geom::CornerRadii;
+
+use crate::{
+    include_labeled_bytes, ButtonBuilder, Color, ControlState, EventCtx, Key, OutlineStyle,
+    ScreenDims, Style, Widget,
+};
+
+#[derive(Clone)]
+pub struct ButtonStyle {
+    pub fg: Color,
+    pub fg_disabled: Color,
+    pub outline: OutlineStyle,
+    pub bg: Color,
+    pub bg_hover: Color,
+    pub bg_disabled: Color,
+}
+
+impl<'a> ButtonStyle {
+    pub fn btn(&self) -> ButtonBuilder<'a> {
+        let base = ButtonBuilder::new()
+            .label_color(self.fg, ControlState::Default)
+            .label_color(self.fg_disabled, ControlState::Disabled)
+            .image_color(self.fg, ControlState::Default)
+            .image_color(self.fg_disabled, ControlState::Disabled)
+            .bg_color(self.bg, ControlState::Default)
+            .bg_color(self.bg_hover, ControlState::Hovered)
+            .bg_color(self.bg_disabled, ControlState::Disabled);
+
+        if self.outline.0 > 0.0 {
+            base.outline(self.outline, ControlState::Default)
+        } else {
+            base
+        }
+    }
+
+    pub fn text(&self, text: &'a str) -> ButtonBuilder<'a> {
```

Comment:
Mostly the changes here were to move methods from `StyledButtons` onto `ButtonStyle`.

So instead of:
```
StyledButtons::btn_outline_text(ctx.style(), "foo");
```

you:
```
ButtonStyle::text(ctx().style().btn_outline, "foo")
```